### PR TITLE
Update 15-manishdpansare.md

### PR DESCRIPTION
**Pull request by manishdpansare**

### CODE REVIEW

File: _slides/15-manishdpansare.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+It is great to be part of the training.
```
Comment (sanofichris) [inline _slides/15-manishdpansare.md:6]: I agree 🤖 🥇

**Review by sanofichris — APPROVED** [review]
awesome

File: _slides/15-manishdpansare.md (hunk 1 above)
Comment (MaheshG2021) [reply _slides/15-manishdpansare.md:6]: 👍

**Review by MaheshG2021 — APPROVED** [review]
Ok got main branch

**Pull request merged**